sdk/src/fee.rs

```diff
@@ -75,15 +88,32 @@ impl FeeStructure {
             .saturating_mul(heap_cost)
     }
 
-    /// Calculate fee for `SanitizedMessage`
     #[cfg(not(target_os = "solana"))]
     pub fn calculate_fee(
```

Comment:
Keep this function live & active. @apfitzge has a good point that most use case just want total_fee. Doesn't need to deprecate it.